examples/token/nep5/nep5.go

```diff
@@ -9,15 +9,15 @@ import (
 // Token holds all token info
 type Token struct {
 	// Token name
-	Name string
+	TName string
```

Comment:
But why?

Reply:
Because `Token` already has `Name` method (which should also be described at manifest). Not sure that it's a good idea. Should we choose another name for this field or for method?

Reply:
Well, I don't think we should have both `Name` exposed and some getter for it, either it's a public field that can be read directly from or this field is to be hidden.

Reply:
We have `Token` defined in the other package (`nep5`), so hiding `TName` requires public setter method for this field. This setter will be used from `token.go` (`tokencontract` package) and, consequently, will be described in token manifest file (as it's a public method). Can we join these packages?

Reply:
> consequently, will be described in token manifest file (as it's a public method). However, it shouldn't be there.

Yep, it shouldn't present in the manifest. Why do we describe internal packages being there? Only functions from the main package should be added.

> Can we join these packages?

That would be a step back as `nep5` is a generic package that can be reused by other tokens. `nep5` methods just shouldn't exist in the manifest.